spt/features/visualizations/imgui/imgui_interface.cpp

```diff
@@ -95,9 +99,13 @@ class SptImGuiFeature : public FeatureWrapper<SptImGuiFeature>
 		InitWin32,
 		InitDx9,
 		OverrideWndProc,
+		FrameSignalCalledAtLeastOnce,
 
 		Loaded = OverrideWndProc,
-	} inline static loadState = LoadState::None;
+	};
+
+	inline static std::atomic<LoadState> loadState = LoadState::None;
+	inline static std::unique_ptr<std::recursive_mutex> imguiMutex = nullptr;
```

Comment:
Is the unique_ptr required here? Wouldn't a normal std::recursive_mutex work as well?

Reply:
You're right. If it's not static it would be required to be moveable since features must be moveable. But since it's static I don't think I need the unique_ptr there.